src/convertToObject.js

```diff
@@ -6,7 +6,35 @@
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  // This onbject will be returned.
+  const styledObject = {};
```

Comment:
[CODE STYLE] Avoid variable namings like `styledObject`. It will decrease code readability. You could use a more descriptive name like `cssProperties`.